boskos/client/client.go

```diff
@@ -129,16 +141,18 @@ func (c *Client) AcquireWait(ctx context.Context, rtype, state, dest string) (*c
 	if ctx == nil {
 		return nil, ErrContextRequired
 	}
+	// request with FIFO priority
+	requestID := uuid.New().String()
 	// Try to acquire the resource until available or the context is
 	// cancelled or its deadline exceeded.
 	for {
-		r, err := c.Acquire(rtype, state, dest)
+		r, err := c.AcquireWithPriority(rtype, state, dest, requestID)
 		if err != nil {
 			if err == ErrAlreadyInUse || err == ErrNotFound {
 				select {
 				case <-ctx.Done():
 					return nil, err
-				case <-time.After(3 * time.Second):
+				case <-time.After(acquireWait):
```

Comment:
Why do we want to send this ten times more often than the TTL requires?

Reply:
I think 10 second would probably work better.